src/js/modules/users/components/IllustratorPage.js

```diff
@@ -5,7 +5,7 @@ import { Grid, Row, Col } from "react-bootstrap/lib";
 
 import { getIllustratorFromSlug } from "../selectors";
 import { getIllustratorArticles } from "../../articles/selectors";
-import { ArticleRow } from "../../articles/components";
+import { ArticleList } from "../../articles/components";
```

Comment:
Find the uses of ArticleRow and replace them all with ArticleList. Then remove ArticleRow from index.